### ISSUE TITLE
Remove interactive mode

### ISSUE THREAD

**Issue by cooperlab** [DigitalSlideArchive/HistomicsTK#622]
Now that the right-click mode has been added, interactive mode is no longer required.

**Comment by manthey** [comment]
Besides highlighting the list when you hover over an annotation element on the image, this also highlights the annotation element on the image when you hover over the entry in the list.  Right-click takes care of the first case, but not the second.  We could add a right-click to the entries in the list to do this, if that is desired.

**Comment by kheffah** [comment]
@manthey I agree that the second case is not taken care by the interactive mode, that's a good point. One thing to fix here, though, is auto-zooming to where the annotation is. Currently, the interactive mode just highlights the annotation but if you're zoomed in on a completely different part of the slide, you don't know were the highlighted annotation is.

**Comment by cooperlab** [comment]
I misunderstood - interactive is still desirable since it may help us locate annotations by clicking them in the list (particularly helpful for rare classes).

If we can pan / zoom to the annotation when it is highlighted in the list then this will make interactive mode very useful.
